docs/.vuepress/components/HomePage.vue

```diff
@@ -155,6 +155,37 @@
           </li>
         </ul>
       </div>
+
+      <div class="intro-block" v-if="isEnglish">
+        <h3>
+          <router-link :to="content['path'] + 'get_involved'">
+            <span class="arrow">→</span>{{content['page-home-section-get_involved-title']}}
+          </router-link>
+        </h3>
+        <ul>
+          <li>
+            <router-link :to="content['path'] + 'get_involved/#1-get-involved-technical'" class="black">
+              {{content['page-home-section-get_involved-item-one']}}
+            </router-link>
+          </li>
+          <li>
+            <router-link :to="content['path'] + 'enterprise/#2-get-involved-research'" class="black">
```

Comment:
Why does this URL start with `enterprise`?

Reply:
I'll write this here, and it also applies to the next two - this was simply a messy error; sorry. I coped the boilerplate from the enterprise page, and it would seem that I forgot to update it. To add to the litany of shame, you'll notice that the link to the section inside the Get Involved Page is labeled wrong - I changed the layout of the page on suggestion before pushing, and missed that I hadn't cleaned up that part of the page. Oops. 😰